content/22.vaccines.md

```diff
@@ -171,7 +171,66 @@ As of December 11, 2020, the United States FDA approved this vaccine under an em
 
 #### COVID-19 Vaccine AstraZeneca
 
-#### Sputnik-V
+#### Sputnik-V (Gam-COVID-Vac and Gam-COVID-Vac-Lyo)
+
+The vaccine Gam-COVID-Vac, nicknamed Sputnik V in reference to the space race, was developed by the Gamaleya National Center of Epidemiology and Microbiology in Moscow. 
+Gamaleya is an organization with prior experience using the adenovirus platform for the development of vaccines for the Middle East respiratory system and Ebola virus, although neither of the previous vaccines were internationally licensed [@url:https://www.who.int/immunization/sage/meetings/2018/october/2_Ebola_SAGE2018Oct_BgDoc_20180919.pdf].
+The development of Sputnik V was financed by the Russian Direct Investment Fund [@doi:ft7j; @doi:ghs3sn]. 
+Sputnik V is a replication deficient recombinant adenovirus vaccine that combines two adenovirus vectors rAd26-S and rAd5-S that express the full-length SARS-CoV-2 spike glycoprotein. 
+However, these vectors are intramuscularly administered individually using two separate vaccines in a prime-boost regimen. 
+The rAd26-S is administered first, followed by rAd5-S 21 days later.
+Both vaccines deliver 10^11 viral particles per dose. 
+This approach is designed to overcome any potential pre-existing immunity to adenovirus in the population [@doi:10.1016/S0140-6736(21)00191-4] as some individuals may possess immunity to Ad5 [@doi:10.1016/j.vaccine.2011.05.025].
+Sputnik V is the only recombinant adenovirus vaccine to utilize two vectors, unlike other vaccines such as the Oxford-AstraZeneca vaccine, which utilizes the chimpanzee adenovirus vector (ChAdOx1 nCoV-19) for both doses [@doi:ghpghz].
+The Sputnik V vaccines are available in both a lyophilized (Gam-COVID-Vac-Lyo) and frozen form (Gan-COVID-Vac), which are stored at 2-8 °C and -18 °C respectively [@doi:gg96hq]. 
```

Comment:
```suggestion
The Sputnik V vaccines are available in both a lyophilized (Gam-COVID-Vac-Lyo) and a frozen form (Gan-COVID-Vac), which are stored at 2 to 8 °C and -18 °C, respectively [@doi:gg96hq]. 
```

Reply:
This won't let me confirm the commit - says it is out of date